app/controllers/api/users_controller.rb

```diff
@@ -8,7 +8,9 @@ def show
 
     def ownership
       @user = get_user
-      render json: @user, serializer: PartnerSerializer
+      balances = TransactionLogEntry.product_balances(@user).to_a
+      r = balances.map{|a, b| {product: ProductShallowSerializer.new(Product.find(a)), coins: b} }
```

Comment:
Line is too long. [98/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space inside { missing.<br>Space inside } missing.